Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM fedora:28
 
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
+ARG LIBVIRT_VERSION=4.9.0
```

Comment:
This will either break eventually or we'll need to keep this up to date.  Instead what we should implement is what is suggested in #9.